src/usr/local/www/system_usermanager_settings.php

```diff
@@ -177,15 +174,19 @@
 
 $section = new Form_Section('Settings');
 
+list($d_hr, $d_min) = [int($g['default_session_timeout_mins']/60), $g['default_session_timeout_mins'] % 60];
```

Comment:
Its not bad ion code. more to the point, if a user scenario leads to a longish timeout, then expressing it in minutes just isn't intuitively clear, and a timeout (like any security setting) should be easily clear on plain reading. I'll lookl at if the code can be simplified though.